packages/subiquity_client/lib/src/client.dart

```diff
@@ -41,14 +41,14 @@ class SubiquityException implements Exception {
 
 class SubiquityClient {
   final _client = HttpClient();
-  late Endpoint _endpoint;
-  late String _host;
```

Comment:
we could possibly remove the `_host` member and use `_endpoint.authority` instead?

Reply:
Yes, we can. It's only used in one place.